lib/travis/model/repository/settings.rb

```diff
@@ -111,6 +116,17 @@ def timeout_log_silence
     value == 0 ? nil : value
   end
 
+  def rate_limit
```

Comment:
I'd suggest this method (and the name of the setting) should include the feature that is being rate limited, because a) we might have other per-repo rate limits in the future, and b) it's more clear without additional context. E.g. `api_builds_rate_limit`